source/io/module_httpClient.js

```diff
@@ -216,8 +216,8 @@
             // withCredentials allows cookies (to be sent / set), HTTP Auth, and TLS Client certs when sending requests Cross Origin
             // Setting to false so communication to servers with Access-Control-Allow-Origin: *
             // See https://w3c.github.io/webappsec-cors-for-developers/#anonymous-requests-or-access-control-allow-origin
-            // TODO mraj: We need a way to expose this configuration to users
-            request.withCredentials = false;
+            // if the jsHttpClientSetCredentials was already called by the user, use that value. Otherwise, default is "false"
+            request.withCredentials = (requestData.withCredentials || false);
```

Comment:
Based on my other comments,, withCredentials should be a property on this class instance so it would be accessed by this._withCredentials. You also would not need the default hard coded here as it would be set in the constructor.